README.md

```diff
@@ -2,6 +2,18 @@
 
 This project acts as a template and exemplar to using Playwright, Typescript and the Page Object Model design pattern to test a multipage application.
 
+
+## Contents
```

Comment:
Can you link to this project from the test repo page?  https://github.com/UKHO/docs/blob/playwright-docs-update/quality-assurance/test-repositories.md

Reply:
Will do :-)